collector/collector.go

```diff
@@ -113,34 +113,38 @@ func deepMergeSlices(sliceA, sliceB []interface{}) ([]interface{}, error) {
 	// Do we need to do the same for other types?
 	firstItem := sliceA[0]
 	if reflect.ValueOf(firstItem).Kind() == reflect.Map {
-		temp := make(map[string]interface{})
-
-		// first we put in temp all the keys present in a, and assign them their existing values
-		for _, item := range sliceA {
-			for k, v := range item.(map[string]interface{}) {
-				temp[k] = v
-			}
-		}
-
-		// then we go through b to merge each of its keys
-		for _, item := range sliceB {
-			for k, v := range item.(map[string]interface{}) {
-				current, ok := temp[k]
-				if ok {
-					// if the key exists, we deep merge it
-					dm, err := DeepMerge(current, v)
-					if err != nil {
-						return []interface{}{}, fmt.Errorf("cannot merge %s with %s", current, v)
-					}
-					temp[k] = dm
-				} else {
-					// otherwise we just set it
-					temp[k] = v
-				}
-			}
-		}
-
-		return []interface{}{temp}, nil
+		fmt.Println("################################## ", firstItem)
+		// temp := make(map[string]interface{})
+		union := append(sliceA, sliceB...)
```

Comment:
just this?

Reply:
yeah, I mean, previously it tried to be smart by doing something like this:

config1
```
- users:
    kairos:
      passwd: kairos
 ```
 
 config2
 ```
 - users:
     kairos:
       ssh_authorized_keys:
         - github: username
```

and produce something like:

```
- users:
     kairos:
       passwd: kairos
       ssh_authorized_keys:
         - github:username
```

But I guess it was silly to do that :facepalm: